### ISSUE TITLE
Warning about deprecated `add-path` command when using the action

### ISSUE THREAD

**Issue by jonasbb** [actions-rs/tarpaulin#11]
Do the checklist before filing an issue:

 * [x] Is this related to the `actions-rs` Actions?
       If you think it's a problem related to Github Actions in general, use GitHub Community forum instead: https://github.community
 * [x] You've read the Contributing section about bugs reporting: https://github.com/actions-rs/.github/blob/master/CONTRIBUTING.md#reporting-bugs
 * [ ] Is this something you can debug and fix? Send a pull request! Bug fixes and documentation fixes are welcome.

## Description

The following warning is printed when using this action.

```
The `add-path` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```
[Source 1](https://github.com/jonasbb/serde_with/runs/1213835493?check_suite_focus=true#step:8:11) | [Source 2](https://github.com/jonasbb/serde_with/actions/runs/291047519)
 
## Action output

```
The `add-path` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

## Expected behavior

No warning about the usage of deprecated utility should be printed.

## Additional context

The Github Changelog explains the reason for this deprecation:
https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

**Comment by jonasbb** [comment]
@svartalf There are only 7 days left to fix this issue before this actions breaks and two (#12 / #14) existing pull requests to fix this problem. It would be great if you could take a look.

https://github.blog/changelog/2020-11-09-github-actions-removing-set-env-and-add-path-commands-on-november-16

**Comment by jonasbb** [comment]
Thanks for merging the PR

**Comment by svartalf** [comment]
As a follow up, it was published as a new `actions-rs/tarpaulin@v0.1` and can be used already. Thanks for PR!
